src/calliope/backend/latex_backend_model.py

```diff
@@ -13,37 +13,37 @@
 from calliope.backend import backend_model, parsing
 from calliope.exceptions import ModelError
 
+if TYPE_CHECKING:
```

Comment:
To avoid circular import errors for imports that are just used for type checking. I think it's handled better in py>3.10, but we'd need to keep this for py3.9 compatability.